src/generators/llms-txt/constants.mjs

```diff
@@ -0,0 +1,3 @@
+export const ENTRY_IGNORE_LIST = ['doc/api/synopsis.md'];
+
+export const LATEST_DOC_API_BASE_URL = 'https://nodejs.org/docs/latest';
```

Comment:
This feels like something that should be stored in a root constants file for visibility